addons/mail/static/src/core/web/discuss_client_action.js

```diff
@@ -1,4 +1,4 @@
-import { Discuss } from "@mail/core/common/discuss";
+import { Discuss } from "@mail/core/public_web/discuss";
```

Comment:
client action is web client only, there is no action manager on public page

Reply:
I plan on removing `discuss_public` and renaming the client action component. We don't really care it is an action, its role is only to restore discuss thread whether it comes from an action or else.

Reply:
Also even if we were to keep `discuss_client_action` in web, this diff is correct: discuss is effectively needed in both web and public and available in the web scope.